content/en/blog/news/2022-gsoc-work-report.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "GSoC 2022 Final Report: Building Jenkins X UI"
+date: 2022-11-13
+draft: false
+description: This project replaces the older UI with the new UI with more powerful features.
+categories: [blog]
+keywords: [Community, jenkins-x, gsoc, 2022]
+slug: "GSoC-2022-work-report"
+aliases: []
+author: Rajat Gupta
+---
+
+## Jenkins X New UI
+It is a helm chart that you can install with the other tools of jx. It is build using [Golang](https://go.dev/) as backend and [Sveltekit](https://kit.svelte.dev/) as frontend, both backend and frontend are build together and are used in the same container.
+
+🌟 It has light and dark themes.
+
+![homepage.png](/images/gsoc-work-report-22/homepage.png)
+
+![pipelinesPage.png](/images/gsoc-work-report-22/pipelinesPage.png)
+
+## Why need a new UI?
+A good UI is essential for a CI/CD tool, as not everyone is familiar with the CLI. The current UI (jx-pipeline-visualizer) is a read-only UI, the user can view the logs of PipelineActivity but neither can start nor stop the pipeline.
+
+Features that UI will provide: 
+- Start and Stop a PipelineActivity.
+- Have a audit trail.
+- A DAG representation of PipelineActivity.
```

Comment:
Replace DAG with "graphical representation" ? It is more general, that it is a DAG is an implementation detail (although important)

Reply:
**DAG** is a popular term but **graphical representation** is the more general term.